src/components/Documentation.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+    <section class="section" id="dochome">
+    <div class="columns is-mobile is-centered">
+      <div class="column is-three-quarters-tablet is-half-desktop has-text-centered">
+        <div class="block">
+          <h1 class="title is-1">Documentation</h1>
+          <p>Resources for using Pangeo Forge</p>
+        </div>
+      </div>
+    </div>
+    <div class="content" id="firststeps">
+        <h3 class="title is-3">First steps</h3>
+        <p>New to Pangeo Forge?  Start here!</p>
+        <ul>
+            <li><strong>What is Pangeo Forge?</strong> - Read more about what Pangeo Forge is and
+            how it works.</li>
+            <li><strong>Introduction Tutorial</strong> - Ready to code?  Walk through creating and
+            deploying your first data transformation pipeline.</li>
+        </ul>
+        <hr class="solid">
+    </div>
+    <div class="content" id="organization">
+        <h3 class="title is-3">How the documentation is organized</h3>
+        <p>There are a number of places to access resources when working with components of Pangeo
+        Forge.  Here is an overview of what you will find:</p>
+        <ul>
+            <li>The <strong>Introduction Tutorial</strong> is the place to start with Pangeo Forge.
```

Comment:
What do you think the Introduction Tutorial should contain? We already have some Jupyter Notebooks rendered to HTML, i.e. the tutorials at https://pangeo-forge.readthedocs.io/en/latest/tutorials/index.html ...

..but my inclination is that we should be steering people away from thinking of notebooks as the way to use `pangeo-forge-recipes`, and more towards developing Python modules, i.e. `recipe.py` files.

So perhaps this should link to a new page called ... https://pangeo-forge.org/docs/intro-tutorial ?

Reply:
I don't know the code well enough to have a notebooks vs. scripts opinion but I trust your intuition, @cisaacstern.

As for content I talk more about that in my comment below, but I think the introduction tutorial should walk someone through deploying a data transformation from start to finish.  The goal is for someone to walk away feeling like "I can deploy data transformations with Pangeo Forge".  A simple but meaningful data goal would be great (maybe geotiff to cog with simple or accessible input/output locations?)